Automattic/sensei
# Add Sensei icon to admin notices

Fixes #4305.

### Changes proposed in this Pull Request

* Adds the Sensei icon to the WC.com connection and WCPC upgrade notices.
* Refactors the WCPC upgrade notice to be consistent with other Sensei notices.

### Testing instructions
- Delete the `sensei_dismiss_wcpc_prompt` and `sensei-wccom-connect-dismissed` options from the database.
- Remove SWCPC from your `plugins` folder (if applicable) and activate WooCommerce.
- Go to _Courses_ > _All Courses_ and ensure the notices appear with the Sensei icon and are styled the same.
- Dismiss both notices. Refresh the page and ensure they don't reappear.

### Screenshot / Video
![Screen Shot 2021-09-15 at 8 05 52 AM](https://user-images.githubusercontent.com/1190420/133433264-92b77487-9825-4d67-bacc-d10eee5a3c21.jpg)
